testng-core-api/testng-core-api-build.gradle.kts

```diff
@@ -15,4 +15,6 @@ dependencies {
     "guiceApi"("com.google.inject:guice")
 
     implementation(projects.testngReflectionUtils)
+    implementation("org.slf4j:slf4j-simple:_")
```

Comment:
I think we should not expose the dependency for the moment and shade it.

Reply:
`api("org.slf4j:slf4j-api:_")` is missing.

Reply:
@juherr - I have fixed this. Please check